src/translator_ingest/ingests/ttd/ttd.py

```diff
@@ -0,0 +1,401 @@
+nameres_url = "https://name-resolution-sri.renci.org/bulk-lookup"  ## DEV instance
+## biolink predicates
+BIOLINK_TREATS = "biolink:treats"
+BIOLINK_STUDIED_TREAT = "biolink:studied_to_treat"
+BIOLINK_PRECLINICAL = "biolink:in_preclinical_trials_for"
+BIOLINK_CLINICAL_TRIALS = "biolink:in_clinical_trials_for"
+## hard-coded mapping of "clinical status" values to biolink "treats" predicates
+## purposely doesn't include all values - rest are filtered out
+clinical_status_map = {
+    ## treats
+    "Approved": BIOLINK_TREATS,
+    "Approved (orphan drug)": BIOLINK_TREATS,
+    "Approved in China": BIOLINK_TREATS,
+    "Approved in EU": BIOLINK_TREATS,
+    "Phase 4": BIOLINK_TREATS,
+    ## studied to treat
+    "Investigative": BIOLINK_STUDIED_TREAT,
+    "Patented": BIOLINK_STUDIED_TREAT,
+    ## in preclinical trials for
+    "Preclinical": BIOLINK_PRECLINICAL,
+    "IND submitted": BIOLINK_PRECLINICAL,
+    ## in clinical trials for
+    "Clinical Trial": BIOLINK_CLINICAL_TRIALS,
+    "Clinical trial": BIOLINK_CLINICAL_TRIALS,
+    "Preregistration": BIOLINK_CLINICAL_TRIALS,
+    "Registered": BIOLINK_CLINICAL_TRIALS,
+    "Phase 0": BIOLINK_CLINICAL_TRIALS,
+    "Phase 1": BIOLINK_CLINICAL_TRIALS,
+    "Phase 1b": BIOLINK_CLINICAL_TRIALS,
+    "Phase 1/2": BIOLINK_CLINICAL_TRIALS,
+    "Phase 1/2a": BIOLINK_CLINICAL_TRIALS,
+    "Phase 1b/2a": BIOLINK_CLINICAL_TRIALS,
+    "Phase 2": BIOLINK_CLINICAL_TRIALS,
+    "Phase 2a": BIOLINK_CLINICAL_TRIALS,
+    "Phase 2b": BIOLINK_CLINICAL_TRIALS,
+    "Phase 2/3": BIOLINK_CLINICAL_TRIALS,
+    "Phase 3": BIOLINK_CLINICAL_TRIALS,
+    "phase 3": BIOLINK_CLINICAL_TRIALS,
+    "NDA filed": BIOLINK_CLINICAL_TRIALS,
+    "BLA submitted": BIOLINK_CLINICAL_TRIALS,
+    "Approval submitted": BIOLINK_CLINICAL_TRIALS,
+}
+## indication names that are known to be problematic - not "conditions that are treated" or I'm worried how the statement will look
+strings_to_filter = [
+    "imaging",
+    "radio",  ## related to imaging
+    "esthesia",  ## for multiple spellings of an(a)esthesia
+    "abortion",  ## problematic? but "spontaneous abortion" aka miscarriage can be treated...
+    "sedation",
+    "Discover",
+    "icide",  ## catches Herbicide, Insecticide, etc. But catches "poisoning" due to these things too
+    "procedure",
+    "barrier",  ## catches Blood brain barrier
+    "astringent",
+    "stimul",  ## catches "Caerulein stimulated..." and ovarian stimulation
+    "suppress",  ## catches Appetite suppressant
+    "contrast",  ## related to imaging
+    "Diagnostic",  ## diagnostic
+    "vasodilator",
+    "Dutch elm disease",  ## this is a plant disease
+    "Exam",
+    "lubricant",
+    "Localisation",
+    "Measur",  ## catches Measure kidney function
+    "Pest attack",  ## plant disease?
+    "Plant grey",  ## catches Plant grey mould disease
+    "Stabil",  ## catches Stabilize muscle contraction
+]
+
+
+## custom functions
+def parse_header(file_path) -> Dict[str, Union[str, int]]:
```

Comment:
thank you for this code comment - very helpful :) I know its a small thing, but I appreciate it!